.travis.yml

```diff
@@ -1,20 +1,25 @@
-language: python
 addons:
   apt:
     packages:
       - libgeoip-dev
 matrix:
   include:
-    - python: "2.7"
-    - python: "3.6"
-    - python: "3.7"
-      dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
-  allow_failures:
-    - python: "3.6"
-    - python: "3.7"
+    - language: python
+      - python: "2.7"
+      - python: "3.6"
+      - python: "3.7"
+    dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
+    allow_failures:
+      - python: "3.6"
+      - python: "3.7"
+    - language: node_js
+      - node_js: "9"
 install:
   - pip install flake8
+  - find . - name \*.py -exec flake8 {} +
```

Comment:
Not a fan of this approach.  Flake8 already knows how to find Python code fast and recursively even in files that do not end in .py.

Reply:
Hey @cclauss: script is incomplete.